SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -216,10 +220,15 @@ void CTurbSSTSolver::Postprocessing(CGeometry *geometry, CSolver **solver_contai
     su2double rho = flowNodes->GetDensity(iPoint);
     su2double mu  = flowNodes->GetLaminarViscosity(iPoint);
 
+    // nijso asks: difference between geometry->nodes->GetWall_Distance and nodes->GetWall_Distance ???
     su2double dist = geometry->nodes->GetWall_Distance(iPoint);
 
-    su2double VorticityMag = GeometryToolbox::Norm(3, flowNodes->GetVorticity(iPoint));
-    VorticityMag = max(VorticityMag, 1e-12); // safety against division by zero
+    // nijso asks: difference between nodes and flownodes ???
```

Comment:
nodes are the turbulent variables, flowNodes are the flow variables.

Reply:
ah of course, thanks!